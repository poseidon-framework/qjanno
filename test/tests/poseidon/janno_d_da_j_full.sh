qjanno "SELECT * FROM d(data/Schiffels_2016,data/Lamnidis_2018,data/Lamnidis_2018_newVersion) GROUP BY source_file HAVING MIN(ROWID) ORDER BY ROWID"
qjanno "SELECT * FROM da(data/Schiffels_2016,data/Lamnidis_2018,data/Lamnidis_2018_newVersion) GROUP BY source_file HAVING MIN(ROWID) ORDER BY ROWID"
qjanno "SELECT * FROM j(data/Schiffels_2016,data/Lamnidis_2018,data/Lamnidis_2018_newVersion) GROUP BY source_file HAVING MIN(ROWID) ORDER BY ROWID"