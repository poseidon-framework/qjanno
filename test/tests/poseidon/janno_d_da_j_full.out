.----------------.-----------------.---------------------------------------------------.-------------.-------------.------------.---------------.---------.---------.---------.
| package_title  | package_version |                    source_file                    | Poseidon_ID | Genetic_Sex | Group_Name |  Publication  | AddCol1 | AddCol2 | AddCol3 |
:================:=================:===================================================:=============:=============:============:===============:=========:=========:=========:
| Lamnidis_2018  | 1.0.1           | data/Lamnidis_2018_newVersion/Lamnidis_2018.janno | XXX011      | M           | POP1       | Lamnidis2018  |         | v2      | v3      |
| Schiffels_2016 | 1.0.1           | data/Schiffels_2016/Schiffels_2016.janno          | XXX001      | M           | POP1       | Schiffels2016 | v1      | v2      |         |
'----------------'-----------------'---------------------------------------------------'-------------'-------------'------------'---------------'---------'---------'---------'
.----------------.-----------------.---------------------------------------------------.-------------.-------------.------------.---------------.---------.---------.---------.
| package_title  | package_version |                    source_file                    | Poseidon_ID | Genetic_Sex | Group_Name |  Publication  | AddCol1 | AddCol2 | AddCol3 |
:================:=================:===================================================:=============:=============:============:===============:=========:=========:=========:
| Schiffels_2016 | 1.0.1           | data/Schiffels_2016/Schiffels_2016.janno          | XXX001      | M           | POP1       | Schiffels2016 | v1      | v2      |         |
| Lamnidis_2018  | 1.0.0           | data/Lamnidis_2018/Lamnidis_2018.janno            | XXX011      | M           | POP1       | Lamnidis2018  |         |         |         |
| Lamnidis_2018  | 1.0.1           | data/Lamnidis_2018_newVersion/Lamnidis_2018.janno | XXX011      | M           | POP1       | Lamnidis2018  |         | v2      | v3      |
'----------------'-----------------'---------------------------------------------------'-------------'-------------'------------'---------------'---------'---------'---------'
.---------------------------------------------------.-------------.-------------.------------.---------------.---------.---------.---------.
|                    source_file                    | Poseidon_ID | Genetic_Sex | Group_Name |  Publication  | AddCol1 | AddCol2 | AddCol3 |
:===================================================:=============:=============:============:===============:=========:=========:=========:
| data/Schiffels_2016/Schiffels_2016.janno          | XXX001      | M           | POP1       | Schiffels2016 | v1      | v2      |         |
| data/Lamnidis_2018/Lamnidis_2018.janno            | XXX011      | M           | POP1       | Lamnidis2018  |         |         |         |
| data/Lamnidis_2018_newVersion/Lamnidis_2018.janno | XXX011      | M           | POP1       | Lamnidis2018  |         | v2      | v3      |
'---------------------------------------------------'-------------'-------------'------------'---------------'---------'---------'---------'
